inference_client/tasks/caption.py

```diff
@@ -19,7 +19,7 @@ class CaptionMixin:
     client: Client
 
     @overload
-    def caption(self, image: Union[str, bytes, 'ArrayType'], **kwargs):
+    def caption(self, *, image: Union[str, bytes, 'ArrayType'], **kwargs):
```

Comment:
revert this change, I like the usage of:
```
t  = model.caption('/path/to/image.jpg')
```

Reply:
Ok. I would like to keep it for now since it is not simple as removing an asterisk. I will raise a separate PR for it.